docs/en-US/component/pagination.md

```diff
@@ -79,6 +79,7 @@ pagination/more-elements
 | default-current-page                | default initial value of current-page, not setting is the same as setting 1                                                     | ^[number]                                                                         | —                                    |
 | layout                              | layout of Pagination, elements separated with a comma                                                                           | ^[string]`string (consists of sizes, prev, pager, next, jumper, ->, total, slot)` | prev, pager, next, jumper, ->, total |
 | page-sizes                          | options of item count per page                                                                                                  | ^[object]`number[]`                                                               | [10, 20, 30, 40, 50, 100]            |
+| append-size-to ^(2.8.3)             | which element the size dropdown appends to                                                                                      | ^[string]                                                                         | —                                    |
```

Comment:
@btea should the version be bumped to `^(2.8.4)` since `^(2.8.3)` was already released https://github.com/element-plus/element-plus/pull/18266?

Reply:
Done